README.adoc

```diff
@@ -249,26 +257,31 @@ You can find more detailed explanations on how to define your own system propert
 them via the `System.getProperty` call in the
 https://openliberty.io/guides/microprofile-intro.html[MicroProfile guide].
 
-=== Testing the processing with JSON-P
-
-Write the `testArtistCount` test case. This test checks that deserialization with JSON-P was done correctly
-and that the correct number of artist is returned.
+=== Testing the binding process
+In order for your test classes to have access to JSON-B you must add the following dependency to your `pom.xml`,
+which has been done for you:
```

Comment:
"In order" doesn't really add to the sentence and can simply be omitted. 

How about something like, "For your test classes to have access to JSON-B, add the following dependency to your `pom.xml` file, which is already done for you:"